src/App.svelte

```diff
@@ -89,20 +95,26 @@
 </main>
 <footer>
   <div class="options">
-    <label>
+    <label class="big-only">
       <input type="checkbox" bind:checked={$onlyMovingo} id="toggleMovingo" />
       Movingo
     </label>
     <label>
-      From
+      <span class="big-only">From</span>
```

Comment:
This will result in the input lacking a label on a small screen both visually and code wise. An input should always have a label. The best thing to do is make it "screen reader only" on smaller screen.

Reply:
Fixed! I'm learning good frontend things here, thanks for reviewing 👍 

Since the inputs that are unique to small screens duplicate some of the big screen inputs (because they are so different or I wanted to move them) some inputs occur twice in the DOM. Will this bother users with screen readers, or is it fine because the duplicated small variants are `display:none` by default?

Reply:
Won´t bother if one of them is display none, that hides it for screen reader even if they are still left in the DOM.